src/test-and-demo-constants.js

```diff
@@ -0,0 +1,101 @@
+export const slides = [
```

Comment:
It would make more sense to make this a .JSON file
I'm not suggesting you change it now, but for future.

Reply:
but my `slideArray` variable references `slides` in the same file. can't use JSON for that.